ci/build_cpp.sh

```diff
@@ -57,21 +51,26 @@ if test "${MAGISTRATE_DOXYGEN_ENABLED:-0}" -eq 1
 then
     MCSS=$PWD/m.css
     GHPAGE=$PWD/DARMA-tasking.github.io
-    git clone --depth=1 "https://${token}@github.com/DARMA-tasking/DARMA-tasking.github.io"
+
+    git clone --depth=1 "https://x-access-token:${TOKEN}@github.com/DARMA-tasking/DARMA-tasking.github.io"
```

Comment:
Shouldn't this be `GITHUB_TOKEN`?
```suggestion
    git clone --depth=1 "https://x-access-token:${GITHUB_TOKEN}@github.com/DARMA-tasking/DARMA-tasking.github.io"
```

Reply:
Good catch! I renamed that token quite few times, and since this part of the code only runs on develop, I wasn't able to notice that :D